documentation/src/docs/asciidoc/user-guide/migration-from-junit4.adoc

```diff
@@ -45,8 +45,8 @@ concurrent test execution. It can be enabled and configured using the following
 <<running-tests-config-params, configuration parameters>>:
 
 `junit.vintage.execution.parallel.enabled=true|false`::
-  Enable/disable parallel execution (defaults to `false`). Requires opt-in for `classes`
-  or `methods` to be executed in parallel using the configuration parameters below.
+  Enable/disable parallel execution (defaults to `false`).
+  Requires opt-in for `classes` or `methods` to be executed in parallel using the configuration parameters below.
```

Comment:
We currently use a different convention, namely hard-wrapping at 90 characters. Please revert this change and reformat your additions accordingly.

Reply:
fixed now